internal/pkg/cli/deploy.go

```diff
@@ -568,7 +573,46 @@ func (o *deployOpts) askNames() error {
 	if err != nil {
 		return fmt.Errorf("select service or job: %w", err)
 	}
-	o.workloadNames = names
+	if len(names) == 1 {
+		o.workloadNames = names
+		return nil
+	}
+
+	// Select workload priority by repeatedly prompting with a multiselect until the list of options is depleted.
+	options := names
+	taggedOptions := make([]string, 0, len(options))
+	currentPriority := 1
+	for len(options) > 0 {
```

Comment:
Should we provide a stop selecting option for them?

Reply:
This only loops over the workloads they previously selected to deploy. There's no path in this logic to say "whoops don't deploy this service," only to say "I don't care when the rest of these are deployed."